codice/ddf
# [2.26.x] updated RTF transformer formatting and now omits null attributes from…

… output

#### What does this PR do?
Updates the RTF Transformers output formatting to include borders around the tables, removes the section breaks between each category, omits null valued attributes or the entire category if no attributes within a category have a value.

#### Who is reviewing it? 
@glenhein 
@jrnorth

#### How should this be tested?
Ingest a some data. Then export the results as RTF via the rest endpoint: 
https://localhost:8993/services/catalog/query?q=*&format=rtf
Verify the formatting is updated and null valued attributes are not present.

#### Checklist:
- [X ] Update / Add Unit Tests

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
